src/Tools/DynamoInstallDetective/Utilities.cs

```diff
@@ -61,10 +61,32 @@ public static IEnumerable LocateDynamoInstallations(string additionalDynamoPath,
         /// version info as Tuple of the file found in the installation based 
         /// on file search pattern. The returned list is sorted based on version 
         /// info.</returns>
+        [Obsolete("Function signature will change in Dynamo 3.0, please refer to FindProductInstallations2")]
         public static IEnumerable FindProductInstallations(string productSearchPattern, string fileSearchPattern)
+        {
+            return FindProductInstallations2(new List<string>() { productSearchPattern }, fileSearchPattern);
```

Comment:
Since the new function has a different parameter type than the old one, it can keep the same name, correct? You don't really need to append the `2` to the name?

Reply:
@mjkkirschner  @aparajit-pratap  Good point, I will keep the original function and remove the obsolete message